package.json

```diff
@@ -7,6 +7,7 @@
   "bugs": "https://github.com/lodash/lodash-cli/issues",
   "repository": "lodash/lodash",
   "license": "MIT",
+  "type": "module",
```

Comment:
Note that this change will break on npm <=3.9.5 (bundled in node <= 6.2.2).

C.f. https://github.com/babel/babel/issues/11000#issuecomment-573807377

Reply:
That's been unsupported for years hasn't it?

Reply:
lodash requires `"node": ">=4.0.0"`, shipping this change implies that we should bump `"node"` because `npm <= 3.9.5` will throw on `"type"` properties in `package.json`.

We could specify `"npm": "> 3.9.5"`, too.

Reply:
npm 3.9.5 is very old. It should be fine to raise the dependency on it in a breaking change, then use the next major for the long-term full ESM conversion

Reply:
As I understand, this change must be applied only in `lodash-es` package. Otherwise it can break usual `lodash` package, because main nodejs program will think, that `lodash` is esm package and will throw error `require is not defined` or something like this

Reply:
Or `lodash` can be converted to dual commonjss/esm package:
https://nodejs.org/dist/latest-v14.x/docs/api/esm.html#esm_dual_commonjs_es_module_packages

Reply:
@rifler or simpler, `lodash` could use semantic versioning as everybody else 🤷🏻‍♂️
If this release brings breaking changes then bump the major (possibly adjusting `engines` accordingly).